### ISSUE TITLE
install error

### ISSUE THREAD

**Issue by ojziff** [saezlab/dorothea#29]
Dear Team,
 
I am struggling installing dorothea from Bioconductor, github or via conda. I get error: package ?dorothea? is not available (for R version 3.6.1)
 
Please could you advise on how I get around this?
 
Many thanks,
Oliver
 
 
```
> BiocManager::install("dorothea")
Bioconductor version 3.10 (BiocManager 1.30.10), R 3.6.1 (2019-07-05)
Installing package(s) 'dorothea'
Old packages: 'rJava', 'Seurat'
Update all/some/none? [a/s/n]:
Update all/some/none? [a/s/n]:
n
Warning message:
package ?dorothea? is not available (for R version 3.6.1)
```
 
```
> devtools::install_github("saezlab/dorothea")
Downloading GitHub repo saezlab/dorothea@HEAD
?  checking for file ?/tmp/slurm_18930860/RtmpkBYJrq/remotes205ac5b742d5b/saezlab-dorothea-6e2b837/DESCRIPTION? (533ms)
?  preparing ?dorothea?:
?  checking DESCRIPTION meta-information ...
?  checking for LF line-endings in source and make files and shell scripts
?  checking for empty or unneeded directories
?  looking to see if a ?data/datalist? file should be added
?  building ?dorothea_1.3.0.tar.gz? (3s)
   Warning: invalid uid value replaced by that for user 'nobody'
   Warning: invalid gid value replaced by that for user 'nobody'
   
ERROR: this R is version 3.6.1, package 'dorothea' requires R >= 4.0
Error: Failed to install 'dorothea' from GitHub:
  (converted from warning) installation of package ?/tmp/slurm_18930860/RtmpkBYJrq/file205ac225b1ee6/dorothea_1.3.0.tar.gz? had non-zero exit status
```

```
> sessionInfo()
R version 3.6.1 (2019-07-05)
Platform: x86_64-conda_cos6-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)
 
Matrix products: default
BLAS/LAPACK: /camp/lab/luscomben/home/users/ziffo/.conda/envs/rtest/lib/libopenblasp-r0.3.7.so
 
locale:
[1] LC_CTYPE=en_GB.UTF-8       LC_NUMERIC=C             
 [3] LC_TIME=en_GB.UTF-8        LC_COLLATE=en_GB.UTF-8   
 [5] LC_MONETARY=en_GB.UTF-8    LC_MESSAGES=en_GB.UTF-8  
 [7] LC_PAPER=en_GB.UTF-8       LC_NAME=C                
 [9] LC_ADDRESS=C               LC_TELEPHONE=C           
[11] LC_MEASUREMENT=en_GB.UTF-8 LC_IDENTIFICATION=C      
 
attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base    
 
loaded via a namespace (and not attached):
[1] BiocManager_1.30.10 compiler_3.6.1      tools_3.6.1 
```

**Comment by christianholland** [comment]
Hi Oliver,

there are two points you must consider:

i) dorothea was initially released with Bioconductor version 3.11 (Currently its Bioconductor version 3.12). Therefore you need to update your Bioconductor version from 3.10 to 3.12.
ii) dorothea requires an R version of at least 4.0.0. So you need to update your R version as well. 

Best wishes,
Christian

**Comment by ojziff** [comment]
Thank you Christian.

It maybe worth considering noting these minimum versions under the README Installation instructions.

Best wishes,
Oliver
